isisd/isis_northbound.c

```diff
@@ -2749,7 +2749,7 @@ const struct frr_yang_module_info frr_isisd_info = {
 		{
 			.xpath = "/frr-isisd:isis/instance",
 			.cbs.create = isis_instance_create,
-			.cbs.delete = isis_instance_delete,
+			.cbs.destroy = isis_instance_delete,
```

Comment:
Ideally we need to rename all callbacks as well to have an `_destroy` suffix instead of ending with `_delete`. We also need to rename `NB_OP_DELETE` to `NB_OP_DESTROY` and make all the corresponding changes (e.g. update `nb_operation_name()`).

Reply:
OK, I think I understand - I'll take a pass through and make those changes...

Reply:
Thanks for the update :)

I think you missed renaming the `_delete()` callbacks from ripd, ripngd and lib/if.c. But to be honest I think it's better this way, otherwise this PR would conflict with the "RIP VRF" PR of mine which is still pending. So please let me rename those callbacks myself later.

This PR looks good to me, except for the `prefixconstptr` problem reported by David. I also have lots of other fixes for C++ (the gRPC northbound plugin is working smoothly already) and I hope this PR can be merged as soon as possible so I can combine my changes with yours.